### PR TITLE
Improved code to make nav section on home page responsive (#1xxup28)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
Uploading Home.mp4…

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/150952985-0e25d6e0-6c03-453c-ab14-716d5028f85d.mp4

File: src/views/Home.vue [hunk 1]
```diff
@@ -215,7 +209,26 @@ ion-card > ion-card-header {
   margin-right: var(--spacer-xl);
 }
 
-main {
-  --page-width: 1024px;
+ion-label[slot="end"] {
```
Comment (dt2patel) [inline src/views/Home.vue]: This css rule is not necessary, find out why you have to use it and what the actual problem is. We are putting ion labels in end slot all the time, and we haven't needed to do this before.

File: src/views/Home.vue [hunk 2]
```diff
@@ -215,7 +209,26 @@ ion-card > ion-card-header {
...
+  nav {
+    display: grid;
+    grid-template-columns: auto 375px;
+    align-items: center;
+    width: var(--page-width);
+    margin: auto;
+  }
+
+  .external {
+    overflow-x: hidden;
```
Comment (dt2patel) [inline src/views/Home.vue]: why do you need to hide overflow in desktop? If you need to reset it why not unset?

**Review by dt2patel — CHANGES_REQUESTED** [review]
See comments

**Comment by dt2patel** [comment]
Additionally, while working on nav section: 

1. remove extra css from this page
2. Add spacing between the internal nav items.
  - on desktop it should add space between depending on parent height
  - on mobile there should be a fixed amount of margin top, about 24px

File: src/views/Home.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/Home.vue]: Done sir

File: src/views/Home.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Home.vue]: I found two things due to which it was happening, first reason was text-align property which was set `initial` and second reason was `flex: 1` was set on slotted label. Then I also went through issue that `Disha` shared in the thread, this issue is solved in 5.7.0 in which they removed flex: 1 property on end slotted label by using `not` selector.
https://github.com/ionic-team/ionic-framework/pull/23807
So as of now I have removed this styling.

**Comment by azkyakhan** [comment]
> Additionally, while working on nav section:
> 
> 1. remove extra css from this page
> 2. Add spacing between the internal nav items.
> 
> * on desktop it should add space between depending on parent height
> * on mobile there should be a fixed amount of margin top, about 24px

1. I haven't added any extra css, I have just adjusted them according to desktop and mobile view.
2. In mobile view I have given a `margin-top` of 24px whereas in desktop view I have unset that margin and justified the content of internal nav evenly so that the spacing between any two adjacent alignment items, before the first alignment item, and after the last alignment item is the same

----
https://user-images.githubusercontent.com/82817616/151307666-9a5956e0-588c-4dcb-ba72-b77177fff9f4.mp4

**Comment by dt2patel** [comment]
For extra CSS, there is some written from targeting anchor tag. Please delete that.

File: src/views/Home.vue (hunk 1 above)
Comment (dt2patel) [reply src/views/Home.vue]: Ok so this will be fixed when upgrade to ionic 6 is complete

**Comment by azkyakhan** [comment]
### The end slot issue of ion-label is solved in Ionic 6
----
localhost_8101_home

**Comment by azkyakhan** [comment]
> For extra CSS, there is some written from targeting anchor tag. Please delete that.

Removed sir

**Review by dt2patel — APPROVED** [review]
looks good

**Pull request merged**